### DISCUSSION TITLE
How to hide IP

### DISCUSSION THREAD

**Discussion by WarpWing** [GitSquared/edex-ui#1033] [Questions]
Hey all! I was wondering if anyone knows how to hide the IP that's shown on the terminal(in the Tron or Ranger themes). I've scoured configurations and unless I missed something, I don't know-how. If I could change it to xx.xxx.xxx.xx or just not showed it at all would be great. I would prefer to replace it with xx.xxx.xxx.xx though if possible. Thank you all :D 

EDIT: Also, is it possible to rewrite some of the commands to do some custom output? I would like to have some custom output for the TYPE and the CPU USAGE parts as well :) Thanks again :D

**Comment by GitSquared** [comment]
There's no configuration to do that unfortunately, you'll have to hack away on the code yourself to add it!

**Reply by WarpWing** [reply]
Hack it I shall! I'll make a PR if I'm successful :)
